### ISSUE TITLE
Warnings for unknown keywords

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#616]
I get lots of these and similar in 1.0.7:

```
attest/docs/deprecated.rst:5: WARNING: unknown keyword: assert
```

The offender is the keyword role, i.e. `:keyword:`assert``.

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/616
- Originally reported by: Dag Odenhall
- Originally created at: 2011-01-23T22:05:11.397

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2011-09-23 06:04:53+00:00_

Fix #616: Allow keywords to be linked via intersphinx.

→ <<cset a7f255d2f69afc0c6a55237c76c6a0ef57a70415>>

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2014-10-18 07:32:19+00:00_

Removing version: 1.0.7 (automated comment)
